src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -19,6 +19,12 @@ $replicasetdetails-border: #ddd;
 $replicasetdetails-table-message: #000;
 $replicasetdetails-table-cell: #777;
 $replicasetdetails-warning: #ffa500;
+$replicasetdetails-message-font-weight: 400;
+
+.kd-replicasetdetail-app-name {
+  color: $muted;
```

Comment:
Please use `$foreground-2`. This is the same as muted. We need to unify those variables.